modules/distribution/src/repository/resources/conf/templates/repository/conf/consent-mgt-config.xml.j2

```diff
@@ -0,0 +1,42 @@
+<!--
+  ~ Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
License header year is wrong. Please fix it.

Reply:
Fixed - https://github.com/wso2/carbon-consent-management/pull/170